build.rs

```diff
@@ -16,12 +16,14 @@
 fn main() {
     println!("cargo:rerun-if-changed=build.rs");
 
-    if version_check::is_min_version("1.59").unwrap_or(false) {
-        println!("cargo:rustc-cfg=usdt_stable_asm");
+    if !version_check::is_min_version("1.59").unwrap_or(false) {
+        println!("cargo:rustc-cfg=usdt_need_asm");
     }
 
     #[cfg(target_os = "macos")]
-    if version_check::is_min_version("1.66").unwrap_or(false) {
-        println!("cargo:rustc-cfg=usdt_stable_asm_sym");
+    if version_check::supports_feature("asm_sym").unwrap_or(false)
+        && !version_check::is_min_version("1.67").unwrap_or(false)
+    {
+        println!("cargo:rustc-cfg=usdt_need_asm_sym");
```

Comment:
if `asm_sym` is supported and the version is 1.67 or greater since the flag is still required in nightly builds prior to those numbered 1.67.

Reply:
I think you mean "version is 1.66 or lower", but I get you.

Reply:
"is NOT 1.67 or greater"